static/js/ducks/telescope.js

```diff
@@ -0,0 +1,26 @@
+import * as API from '../API';
```

Comment:
This file should be named telescopes.js for consistency (see other ducks files for examples)